movai_core_shared/common/time.py

```diff
@@ -0,0 +1,32 @@
+from datetime import datetime
+
+def current_time_string() -> str:
+    return datetime.now().strftime("%d/%m/%Y at %H:%M:%S")
+
+def current_timestamp_float() -> float:
+    return datetime.now().timestamp()
+
+def current_timestamp_int() -> int:
+    return int(datetime.now().timestamp())
+
+def delta_time_int(delta: int) -> int:
+        """returns a future time in timestamp format.
+
+        Args:
+            expiration_delta (int): the time delta from now.
+
+        Returns:
+            int: an int representing the time delta.
+        """
+        return int((datetime.now() + delta).timestamp())
+
+def delta_time_float(delta: int) -> float:
```

Comment:
why create int and float for same function ? and if we must do it then it should be one function for the implementation (because it's the same functionality) and the other 2 functions will do the convert